src/divide.js

```diff
@@ -17,9 +17,6 @@ var _curry2 = require('./internal/_curry2');
  *
  *      R.divide(71, 100); //=> 0.71
  *
- *      var half = R.divide(R.__, 2);
- *      half(42); //=> 21
- *
```

Comment:
I think we'd still want to show how to do this, either with `flip` or with `partial`.

(Unless we ever decide to add things like `divideBy` and cross-reference the documentation.)